docs/modules/deployment/pages/minion/message-broker/kafka.adoc

```diff
@@ -52,19 +52,38 @@ config:property-set bootstrap.servers my-kafka-ip-1:9092,my-kafka-ip-2:9092<1>
 config:update
 ----
 
-<1> Connect to the specified Kafka nodes and adjust the IPs or FQDNs with the Kafka port (9092) accordingly.
+<1> Adjust the IPs or FQDNs and the Kafka port (9092) to match your broker configuration.
 
-If you set more than one Kafka node as `bootstrap.servers`. the driver attempts to connect to the first entry.
+If you set more than one Kafka node as `bootstrap.servers` the driver attempts to connect to the first entry.
 If that is successful, the whole broker topology will be discovered and will be known by the client.
 The other entries are used only if the connection to the first entry fails.
 
-IMPORTANT: Ensure you use the FQDN or IP for your Kafka nodes as configured as advertised listener.
+Any valid Kafka configuration property can be used.
+      An example using SASL/SCRAM with TLS:
 
-NOTE: You can still configure module-specific config for sink IPC at `org.opennms.core.ipc.sink.kafka.cfg`; similarly for RPC and twin.
-      Module-specific config takes precedence over common config specified on `org.opennms.core.ipc.kafka.cfg`.
+[source, karaf]
+----
+config:edit org.opennms.core.ipc.kafka
+config:property-set bootstrap.servers my-kafka-ip-1:9096,my-kafka-ip-2:9096
+config:property-set security.protocol=SASL_SSL
+config:property-set sasl.mechanism SCRAM-SHA-512
+config:property-set sasl.jaas.config 'org.apache.kafka.common.security.scram.ScramLoginModule required username="opennms-ipc" password="kafka";'
+config:update
+----
+
+NOTE: You can configure module-specific values for modules `sink`, `rpc`, and `twin` by replacing PID `org.opennms.core.ipc.kafka` in the `config:edit` command with `org.opennms.core.ipc.sink.kafka`, `org.opennms.core.ipc.rpc.kafka`, and `org.opennms.core.ipc.twin.kafka`, respectively.
+      Module-specific config takes precedence over common config specified in config for PID `org.opennms.core.ipc.kafka`.
```

Comment:
```suggestion
Module-specific configuration takes precedence over common configuration specified in config for PID `org.opennms.core.ipc.kafka`.
```
I'm not clear what "specified in config for PID" means. Should it be in "config:edit" command?

Reply:
Yeah, I wasn't sure how to phrase this. PID stands for Persistent Identity; it's a term from the OSGi bundle configuration standard, meaning the config file for a specific OSGi bundle. It's a useful shortcut for the handful of users like me who know what it is, but probably not great for a general audience. I'll rework it to remove that reference.